sleep/sleep.go

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright (c) 2021 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"time"
```

Comment:
if we use nanoseconds or ticks, do we need the time package?

Reply:
It's an internal package